server/crawler/index.ts

```diff
@@ -196,9 +204,19 @@ export default async function getCralwerData(url: string) {
   let body = ''
   let ogImage = ''
   let images:any = []
+  let browser:any
+  let page:any
+  let content:any
 
   try {
-    const { data: content } = await axios.get(encodeURI(url as string))
+    await axios.get(encodeURI(url as string),{ withCredentials: true }).then((element)=>{
+      content = element.data
+    }).catch(async () => {
+        ({ browser } = await getBrowserPage());
+        page = await browser.newPage();
+        await page.goto(encodeURI(url as string), {'timeout': 90000, waitUntil: 'networkidle2' });
```

Comment:
Please set default timeout for navigation and resources
Also please reduce timeout to 10s

Reply:
Default is 30
But I just tried to get some news URLs 30 seconds is not enough

Reply:
headless, tried 10 seconds is still error, but default 30 is ok